targets/TARGET_Samsung/security_subsystem/drivers/mb_cmd_rng.c

```diff
@@ -0,0 +1,166 @@
+/****************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/*************** Include Files ********************************************/
+/* Driver Common */
+#include "sss_common.h"
+#include "sss_driver_util.h"
+#include "sss_map.h"
+#include "sss_driver_error.h"
+#include "mb_cmd_system.h"
+
+/* Driver Algorithm */
+#include "mb_cmd_rng.h"
+
+/*************** Assertions ***********************************************/
+
+/*************** Definitions / Macros *************************************/
+
+/*************** New Data Types (Basic Data Types)  ***********************/
+
+/*************** New Data Types *******************************************/
+
+/*************** Constants ************************************************/
+
+/*************** Variable declarations ************************************/
+
+/*************** Functions  ***********************************************/
+
+/*! @fn     mb_generate_random_number(unsigned int *random_number, unsigned int random_wlen)
+ *  @ingroup    SECURITY_SSS_MAILBOX
+ *  @brief      mailbox api for random number generation
+ *  @version    v0.50 : 2016.8.13 Init. release version
+ *  @version    v0.51 : 2018.01.13 Modifyh for S111 SRAM code
```

Comment:
Is "Modifyh" correct? What does that mean? Should this be "Modified"?

Reply:
Thanks for finding this out. The Samsung team changed the code and I `grep -rin "modifyh" *` and found nothing of such.